src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -37,7 +37,7 @@ export class ArtworkSidebarCommercial extends React.Component<
 
   render() {
     const { artwork } = this.props
-    if (!artwork.sale_message && !artwork.is_inquireable) {
+    if (!artwork.sale_message) {
```

Comment:
I think you might still need to check if the work is inquireable to account for the case of artworks where availability is hidden and legacy artworks with 'not for sale' availability that have no sale_message but still should have 'Contact' button.